runner.go

```diff
@@ -191,8 +191,14 @@ func handleCommand(hndlr *cmdHandler) (int, []byte, float64, error) {
 	}
 
 	// emit the metric for how long it took us and return code
-	hndlr.gs.Timing(fmt.Sprintf("%v.time", hndlr.opts.Label), wallRtMs, nil)
-	hndlr.gs.Gauge(fmt.Sprintf("%v.exit_code", hndlr.opts.Label), float64(ret), nil)
+	tags := []string{}
+
+	if len(hndlr.opts.Group) > 0 {
+		tags = append(tags, fmt.Sprintf("cronner_group:%v", hndlr.opts.Group))
```

Comment:
I've forked `cronner` to make sure the project is still getting love, and I'm looking to merge this in to my fork. This PR looks great, minus one thing: can you change this to a `%s`?

I know it I used `%v` in the other place that's similar to this, which leads me to believe I had no idea what the heck I was thinking when I wrote that. 😂

If you have the time, would you mind converting the other one too? If not, no biggie and I'll clean it up later:

* https://github.com/PagerDuty/cronner/blob/15739369fb7dd06c5a1de0295504eed09a9d25f2/runner.go#L281